### PR TITLE
Remove base_url, add external_url & internal_url

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

This PR updates the documentation for the parent PR.

Removes the use of `base_url` from our documentation, as it is deprecated.
Insert `external_url` and `internal_url` core configuration as the replacements.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/35224
- Link to frontend pull request: https://github.com/home-assistant/frontend/pull/5755
- Link to JavaScript WebSocket pull request: https://github.com/home-assistant/home-assistant-js-websocket/pull/115
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/ecosystem/certificates/lets_encrypt.markdown [hunk 1]
```diff
@@ -333,7 +333,6 @@ Now edit your `configuration.yaml` file to reflect the SSL entries and your base
 http:
   ssl_certificate: /etc/letsencrypt/live/examplehome.duckdns.org/fullchain.pem
   ssl_key: /etc/letsencrypt/live/examplehome.duckdns.org/privkey.pem
-  base_url: examplehome.duckdns.org
```
Comment (balloob) [inline source/_docs/ecosystem/certificates/lets_encrypt.markdown:336]: The text above mentions base url. We should probabably add a line to change that in your config.

Comment (frenck) [reply source/_docs/ecosystem/certificates/lets_encrypt.markdown:336]: Well spotted, thanks! Will update

File: source/_docs/ecosystem/haproxy.markdown [hunk 2]
```diff
@@ -96,8 +96,6 @@ In your `configuration.yaml` file, edit the [HTTP component](/integrations/http/
 http:
   # For extra security set this to only accept connection on localhost if HAProxy is on the same machine
   # server_host: 127.0.0.1
-  # Update this line to be your domain
-  base_url: https://example.com
```
Comment (balloob) [inline source/_docs/ecosystem/haproxy.markdown:100]: Probably also should have a sentence about setting external url.

File: source/_docs/ecosystem/nginx.markdown [hunk 3]
```diff
@@ -89,8 +89,6 @@ On your `configuration.yaml` file, edit the `http` component.
 http:
   # For extra security set this to only accept connections on localhost if NGINX is on the same machine
   # server_host: 127.0.0.1
-  # Update this line to be your domain
```
Comment (balloob) [inline source/_docs/ecosystem/nginx.markdown:92]: Should we also mention here about setting external url.

**Review by balloob — DISMISSED** [review]

**Comment by frenck** [comment]
Done!

**Pull request merged**